packages/protocol/contracts/governance/Governance.sol

```diff
@@ -1357,7 +1257,7 @@ contract Governance is
    * @return Hotfix executed.
    * @return Hotfix exection time limit.
    */
-  function getL2HotfixRecord(bytes32 hash) public view onlyL2 returns (bool, bool, bool, uint256) {
+  function getL2HotfixRecord(bytes32 hash) public view returns (bool, bool, bool, uint256) {
```

Comment:
Is there a backwards compatibility concern with renaming this to `getHotfixRecord` and/or adding a `getHotfixRecord` function on top of this one?

Reply:
I was thinking there is still a chance this would conflict withou outdated tooling from 3rd parties, but since the L1 version of this function is now gone, the outcome would be the same.

Fixed